addon/mixins/axis-titles.js

```diff
@@ -0,0 +1,181 @@
+  /**
+   * Title for the y axis
+   * @type {String}
+   */
+  yValueDisplayName: null,
+
+  /**
+   * Computed title for the x axis, if the `hasAxisTitles` boolean is false
+   * `xAxisTitleDisplayValue` should be an empty string
+   * @type {String}
+   */
+  xAxisTitleDisplayValue: Ember.computed('hasAxisTitles', 'xValueDisplayName', function(){
+    return this.get('hasAxisTitles') === true ? this.get('xValueDisplayName') : '';
+  }),
+
+  /**
+   * Computed title for the x axis, if the `hasAxisTitles` boolean is false
+   * `yAxisTitleDisplayValue` should be an empty string
+   * @type {String}
+   */
+  yAxisTitleDisplayValue: Ember.computed('hasAxisTitles', 'yValueDisplayName', function(){
+    return this.get('hasAxisTitles') === true ? this.get('yValueDisplayName') : '';
+  }),
+
+  /**
+   * Computed left margin, if the `hasAxisTitles` is false this should be 0
+   * @type {Number}
+   */
+  horizontalMarginLeft: Ember.computed('hasAxisTitles', function(){
+    return this.get('hasAxisTitles') === true ? 20 : 0;
+  }),
+
+  /**
+   * Accessed value within d3
+   * @alias `horizontalMarginLeft`
+   * @type {Number}
+   */
+  marginLeft: Ember.computed.alias('horizontalMarginLeft'),
+
+  /**
+   * Computed title height plus label padding or 0 if `hasAxisTitles` is false
+   * @type {Number}
+   */
+  axisTitleHeightOffset: Ember.computed('axisTitleHeight', 'labelPadding', function() {
+    if(this.get('hasAxisTitles') === true){
+      return this.get('axisTitleHeight') + this.get('labelPadding');
+    }else{
+      return 0;
+    }
+  }),
+
+  /**
+   * D3 reference to x axis-title
+   * @type {Object}
+   */
+  xAxisTitle: Ember.computed(function() {
+    return this.selectOrCreateAxisTitle('.x.axis-title').attr('class', 'x axis-title');
+  }).volatile(),
+
+  /**
+   * D3 reference to y axis-title
+   * @type {Object}
+   */
+  yAxisTitle: Ember.computed(function() {
+    return this.selectOrCreateAxisTitle('.y.axis-title').attr('class', 'y axis-title');
+  }).volatile(),
+
+  /**
+   * Amount of padding for the axis, `labelHeightOffset` + `labelPadding`
+   * @type {Number}
+   */
+  axisPadding: Ember.computed(function(){
+    return this.get('labelHeightOffset') + this.get('labelPadding');
+  }).property('labelHeightOffset', 'labelPadding'),
+
+  /**
+   * Position of x axis title on the x axis
+   * @type {Number}
+   */
+  xAxisPositionX: Ember.computed(function(){
+    return this.get('graphicWidth') / 2 + this.get('labelWidthOffset');
+  }).property('graphicWidth', 'labelWidthOffset'),
+
+  /**
+   * Position of x axis title on the y axis
+   * @type {Number}
+   */
+  xAxisPositionY: Ember.computed(function(){
+    return this.get('graphicBottom') + this.get('axisPadding');
+  }).property('graphicBottom', 'axisPadding'),
+
+  /**
+   * Position of y axis title on the x axis
+   * @type {Number}
+   */
+  yAxisPositionX: Ember.computed(function(){
+    return -1 * (this.get('graphicHeight') / 2 + this.get('labelWidthOffset'));
```

Comment:
This can just be `-(this.get('graphicHeight') / 2 + this.get('labelWidthOffset'))`